### PR TITLE
alpine: add 3.9

### PR DESCRIPTION
**Pull request by ncopa**

### CODE REVIEW

File: library/alpine [hunk 1]
```diff
@@ -4,119 +4,140 @@
...
 i386-GitFetch: refs/heads/rootfs/library-3.8/x86
-i386-GitCommit: e8dbc6e8cafd058246d3dd686156f30ec8a5ce89
+i386-GitCommit: 197f1e39f11e16d1eb06dfed7452b7b1e3674a9e
 i386-Directory: versions/library-3.8/x86
 amd64-GitFetch: refs/heads/rootfs/library-3.8/x86_64
-amd64-GitCommit: d39bc7743955e37ac15e8d82af14410c0b8b9a03
+amd64-GitCommit: 63a4b585539aec349a3b1b5109be7bfece856cff
 amd64-Directory: versions/library-3.8/x86_64
 
+Tags: 3.9
```
Comment (tianon) [inline library/alpine]: Did you want to move `latest` too since 3.9 is GA now?

Comment (ncopa) [reply library/alpine]: Yes, but I will have to do that later. I dont have time today.

File: library/alpine [hunk 2]
```diff
@@ -4,119 +4,140 @@
...
+Architectures: arm64v8, arm32v6, ppc64le, s390x, i386, amd64
```
Comment (J0WI) [inline library/alpine]: Will there be support for `armv7`?

File: library/alpine (hunk 1 above)
Comment (tianon) [reply library/alpine]: No worries, I'll push a commit to update it here and we can fix the generate script separately. :+1:

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index a810988..8a9550b 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -15,6 +15,12 @@ alpine:3.7 @ arm64v8
 alpine:3.7 @ i386
 alpine:3.7 @ ppc64le
 alpine:3.7 @ s390x
+alpine:3.8 @ amd64
+alpine:3.8 @ arm32v6
+alpine:3.8 @ arm64v8
+alpine:3.8 @ i386
+alpine:3.8 @ ppc64le
+alpine:3.8 @ s390x
 alpine:edge @ amd64
 alpine:edge @ arm32v6
 alpine:edge @ arm64v8
diff --git a/_bashbrew-list b/_bashbrew-list
index cb2c82f..6305ea0 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -6,5 +6,6 @@ alpine:3.5
 alpine:3.6
 alpine:3.7
 alpine:3.8
+alpine:3.9
 alpine:edge
 alpine:latest
diff --git a/alpine_3.1/rootfs.tar.xz b/alpine_3.1/rootfs.tar.xz
index a8ec4fb..3c5218d 100644
Binary files a/alpine_3.1/rootfs.tar.xz and b/alpine_3.1/rootfs.tar.xz differ
diff --git a/alpine_3.2/rootfs.tar.xz b/alpine_3.2/rootfs.tar.xz
index 9b91c56..ccda6cc 100644
Binary files a/alpine_3.2/rootfs.tar.xz and b/alpine_3.2/rootfs.tar.xz differ
diff --git a/alpine_3.3/rootfs.tar.xz b/alpine_3.3/rootfs.tar.xz
index c64cf7e..c4f2298 100644
Binary files a/alpine_3.3/rootfs.tar.xz and b/alpine_3.3/rootfs.tar.xz differ
diff --git a/alpine_3.4/rootfs.tar.xz b/alpine_3.4/rootfs.tar.xz
index 16d8067..8dcb6e0 100644
Binary files a/alpine_3.4/rootfs.tar.xz and b/alpine_3.4/rootfs.tar.xz differ
diff --git a/alpine_3.5/rootfs.tar.xz b/alpine_3.5/rootfs.tar.xz
index 8e2c853..0d6a61f 100644
Binary files a/alpine_3.5/rootfs.tar.xz and b/alpine_3.5/rootfs.tar.xz differ
diff --git a/alpine_3.6/rootfs.tar.xz b/alpine_3.6/rootfs.tar.xz
index a2a3b77..b61ddad 100644
Binary files a/alpine_3.6/rootfs.tar.xz and b/alpine_3.6/rootfs.tar.xz differ
diff --git a/alpine_3.7/rootfs.tar.xz b/alpine_3.7/rootfs.tar.xz
index d69b3b7..bc515a2 100644
Binary files a/alpine_3.7/rootfs.tar.xz and b/alpine_3.7/rootfs.tar.xz differ
diff --git a/alpine_3.1/Dockerfile b/alpine_3.8/Dockerfile
similarity index 100%
copy from alpine_3.1/Dockerfile
copy to alpine_3.8/Dockerfile
diff --git a/alpine_3.3/rootfs.tar.xz b/alpine_3.8/rootfs.tar.xz
similarity index 20%
copy from alpine_3.3/rootfs.tar.xz
copy to alpine_3.8/rootfs.tar.xz
index c64cf7e..ed4cc00 100644
Binary files a/alpine_3.3/rootfs.tar.xz and b/alpine_3.8/rootfs.tar.xz differ
diff --git a/alpine_latest/rootfs.tar.xz  'tar -t' b/alpine_3.8/rootfs.tar.xz  'tar -t'
similarity index 100%
copy from alpine_latest/rootfs.tar.xz  'tar -t'
copy to alpine_3.8/rootfs.tar.xz  'tar -t'
diff --git a/alpine_edge/rootfs.tar.xz b/alpine_edge/rootfs.tar.xz
index eb74ca5..a394961 100644
Binary files a/alpine_edge/rootfs.tar.xz and b/alpine_edge/rootfs.tar.xz differ
diff --git a/alpine_edge/rootfs.tar.xz  'tar -t' b/alpine_edge/rootfs.tar.xz  'tar -t'
index 732dc19..184fc4c 100644
--- a/alpine_edge/rootfs.tar.xz  'tar -t'	
+++ b/alpine_edge/rootfs.tar.xz  'tar -t'	
@@ -142,6 +142,7 @@ etc/services
 etc/shadow
 etc/shells
 etc/ssl/
+etc/ssl/cert.pem
 etc/ssl/certs/
 etc/ssl/ct_log_list.cnf
 etc/ssl/ct_log_list.cnf.dist
diff --git a/alpine_latest/rootfs.tar.xz b/alpine_latest/rootfs.tar.xz
index 5629761..93dc6e4 100644
Binary files a/alpine_latest/rootfs.tar.xz and b/alpine_latest/rootfs.tar.xz differ
diff --git a/alpine_latest/rootfs.tar.xz  'tar -t' b/alpine_latest/rootfs.tar.xz  'tar -t'
index 3d853fd..184fc4c 100644
--- a/alpine_latest/rootfs.tar.xz  'tar -t'	
+++ b/alpine_latest/rootfs.tar.xz  'tar -t'	
@@ -144,8 +144,15 @@ etc/shells
 etc/ssl/
 etc/ssl/cert.pem
 etc/ssl/certs/
+etc/ssl/ct_log_list.cnf
+etc/ssl/ct_log_list.cnf.dist
+etc/ssl/misc/
+etc/ssl/misc/CA.pl
+etc/ssl/misc/tsget
+etc/ssl/misc/tsget.pl
 etc/ssl/openssl.cnf
-etc/ssl/x509v3.cnf
+etc/ssl/openssl.cnf.dist
+etc/ssl/private/
 etc/sysctl.conf
 etc/sysctl.d/
 etc/sysctl.d/00-alpine.conf
@@ -162,12 +169,8 @@ lib/apk/db/triggers
 lib/firmware/
 lib/ld-musl-x86_64.so.1
 lib/libc.musl-x86_64.so.1
-lib/libcrypto.so.43
-lib/libcrypto.so.43.0.1
-lib/libssl.so.45
-lib/libssl.so.45.0.1
-lib/libtls.so.17
-lib/libtls.so.17.0.1
+lib/libcrypto.so.1.1
+lib/libssl.so.1.1
 lib/libz.so.1
 lib/libz.so.1.2.11
 lib/mdev/
@@ -176,6 +179,7 @@ media/cdrom/
 media/floppy/
 media/usb/
 mnt/
+opt/
 proc/
 root/
 run/
@@ -389,12 +393,14 @@ usr/bin/xxd
 usr/bin/xzcat
 usr/bin/yes
 usr/lib/
-usr/lib/libcrypto.so.43
-usr/lib/libcrypto.so.43.0.1
-usr/lib/libssl.so.45
-usr/lib/libssl.so.45.0.1
-usr/lib/libtls.so.17
-usr/lib/libtls.so.17.0.1
+usr/lib/engines-1.1/
+usr/lib/engines-1.1/afalg.so
+usr/lib/engines-1.1/capi.so
+usr/lib/engines-1.1/padlock.so
+usr/lib/libcrypto.so.1.1
+usr/lib/libssl.so.1.1
+usr/lib/libtls-standalone.so.1
+usr/lib/libtls-standalone.so.1.0.0
 usr/local/
 usr/local/bin/
 usr/local/lib/
```

</details>

**Comment by yosifkit** [comment]
Build test of #5367; 6abeba63; `amd64` (`alpine`):

```console

$ bashbrew build alpine:3.6
Building bashbrew/cache:8c7b735103bf17a558039590b36e0fe00780b39a00c2e93734061d4dfbe39d6e (alpine:3.6)
Tagging alpine:3.6

$ test/run.sh alpine:3.6
testing alpine:3.6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:3.7
Building bashbrew/cache:74790424dcee136732353d65b8fc4fe416f7a64fbcd6411fe868ad63cc31dc0a (alpine:3.7)
Tagging alpine:3.7

$ test/run.sh alpine:3.7
testing alpine:3.7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:3.8
Building bashbrew/cache:e15e98bd72d44c21c9118945c05d1d75056451447ce9cf37e5acdd3845f436f0 (alpine:3.8)
Tagging alpine:3.8

$ test/run.sh alpine:3.8
testing alpine:3.8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:3.9
Building bashbrew/cache:ccdafb67a2429c6141d7881f16a96c8afcea9d950d95aa3d6934af56d4f6ccd5 (alpine:3.9)
Tagging alpine:3.9
Tagging alpine:latest

$ test/run.sh alpine:3.9
testing alpine:3.9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:edge
Building bashbrew/cache:36e83184a93418b59d676f235d161ce4d89c282e1b9827c304a5afa99ec4e05a (alpine:edge)
Tagging alpine:edge

$ test/run.sh alpine:edge
testing alpine:edge
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:3.1
Building bashbrew/cache:51cad26888c52575528e318ba940c84a036d6a7686779c62c0d3bbc958301930 (alpine:3.1)
Tagging alpine:3.1

$ test/run.sh alpine:3.1
testing alpine:3.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:3.2
Building bashbrew/cache:7e969b8af59cb5433a9aee34e1050ae30593790209e320b436be32f9be8dafb5 (alpine:3.2)
Tagging alpine:3.2

$ test/run.sh alpine:3.2
testing alpine:3.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:3.3
Building bashbrew/cache:4d8d83c7a65cd8bdf3008337ce99b8c8a30bb1c438bc417d55250c7201df0328 (alpine:3.3)
Tagging alpine:3.3

$ test/run.sh alpine:3.3
testing alpine:3.3
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:3.4
Building bashbrew/cache:2ed5b187de7dc992c776f4a1dbbc2b1c64a5dc36467185a1c83b943fbb1eb7e3 (alpine:3.4)
Tagging alpine:3.4

$ test/run.sh alpine:3.4
testing alpine:3.4
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:3.5
Building bashbrew/cache:0e30413975fcafeb5c67fc94f80313d135cabbc1855349036e473824ac564e2c (alpine:3.5)
Tagging alpine:3.5

$ test/run.sh alpine:3.5
testing alpine:3.5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**